pkg/apis/jaegertracing/v1/jaeger_types.go

```diff
@@ -378,7 +378,7 @@ type JaegerDependenciesSpec struct {
 	TTLSecondsAfterFinished *int32 `json:"ttlSecondsAfterFinished,omitempty"`
 
 	// +optional
-	Resources v1.ResourceRequirements `json:"resources,omitempty"`
```

Comment:
This breaks compatibility with CRs specifying that node, doesn't it? Would it be possible to deprecate first, and use the value from the CR to populate the new property?

Reply:
The `commonSpec.Resources` is a direct replacement as the `CommonSpec` is inline. So from the CR's perspective there is no change.